yarn-project/sequencer-client/src/sequencer/setup_phase_manager.ts

```diff
@@ -45,6 +45,22 @@ export class SetupPhaseManager extends AbstractPhaseManager {
       );
     tx.unencryptedLogs.addFunctionLogs(newUnencryptedFunctionLogs);
     await this.publicStateDB.checkpoint();
-    return { publicKernelOutput, publicKernelProof, revertReason, returnValues: undefined };
+
+    // Return a list of setup proving requests
+    const kernelRequests = kernelInputs.map(input => {
+      const request: PublicKernelRequest = {
+        type: PublicKernelType.SETUP,
+        inputs: input,
+      };
+      return request;
+    });
+    return {
+      kernelRequests,
+      kernelInputs,
+      publicKernelOutput,
+      publicKernelProof,
```

Comment:
this `publicKernelProof` is dead code at this point right?

Reply:
Yes, I'll remove once I have everything working in the prover